google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ClientSideStatementValueConverters.java

```diff
@@ -382,8 +382,9 @@ public Class<PgTransactionMode> getParameterClass() {
     public PgTransactionMode convert(String value) {
       PgTransactionMode mode = new PgTransactionMode();
       // Transaction mode may contain multiple spaces.
+      String valueWithoutNotDeferrable = value.replaceAll("(?i)(not\\s+deferrable)", " ");
```

Comment:
nit: please check for a better variable name for `valueWithoutNotDeferrable`, the double negative makes it slightly confusing to read.